### PR TITLE
fix: #51 trapper perks

### PR DESCRIPTION
**Pull request by Viou**
I'm not sure about some of the text for the traps; e.g. should I be using:

-  %game.action.attack% 1
-  +1 %game.action.attack% 
-  %game.damage:1% 

I ended up using %game.action.attack% 1
I'm also not sure what the value of value property should be for custom attackModifier.effects.

Lastly, I noticed the perks for this one are so long that you can't read the last one. So heads up, that might need a style fix.

### CODE REVIEW

**Comment by Lurkars** [comment]
Sadly I don't know the real design. But anyways, the value should be what's written on the card. The custom field should only be used, when there is a short cut for the value and a longer text on the perks. For now your have mostly used custom everywhere and for example the heal 2 trap card just have a value of "+2 heal", which will never indicate in the card itself, that you need to create a trap instead of performing a heal action. For now I would asked you to just use the value field for all and ignore the custom field. I will polish this later, when I got a view on the actual cards and adjusting design. For now it will work better when the whole text is also visible on the card!

**Comment by Viou** [comment]
I'm not sure what you mean by removing "custom". For custom effects, I've moved "hint" into "value". If that's not what you're looking for, could you provide an example of one of these changes? 

This is what the change looks like locally
<details>
  <summary>Spoiler</summary>
  image
</details>

**Comment by Viou** [comment]
Alright, I've cleaned up the PR

File: data/fh/character/trap.json [hunk 1]
```diff
@@ -91,6 +91,155 @@
...
+          }
+        },
+        {
+          "count": 1,
+          "attackModifier": {
+            "type": "plus0",
+            "effects": [
+              {
+                "type": "custom",
+                "value": "Create one %game.action.attack% 1 trap in an empty hex adjacent to the target"
```
Comment (Lurkars) [inline data/fh/character/trap.json:148]: Replace `%game.action.attack% 1` with `%game.damage:1%`

File: data/fh/character/trap.json [hunk 2]
```diff
@@ -91,6 +91,155 @@
...
+    },
+    {
+      "type": "custom",
+      "count": 1,
+      "custom": "Ignore scenario effects"
+    },
+    {
+      "type": "custom",
+      "count": 1,
+      "custom": "Whenever you long rest, you may create one %game.action.attack% 1 trap in an adjacent empty hex"
```
Comment (Lurkars) [inline data/fh/character/trap.json:231]: Replace `%game.action.attack% 1` with `%game.damage:1%`

File: data/fh/character/trap.json [hunk 3]
```diff
@@ -91,6 +91,155 @@
...
+    },
+    {
+      "type": "custom",
+      "count": 1,
+      "custom": "Whenever you enter a hex with a trap, you may choose to not spring the trap"
+    },
+    {
+      "type": "custom",
+      "count": 1,
+      "custom": "At the start of each scenario, you may create one %game.action.attack% 2 trap in an adjacent empty hex"
```
Comment (Lurkars) [inline data/fh/character/trap.json:241]: Replace `%game.action.attack% 2` with `%game.damage:2%`

**Review by Lurkars — CHANGES_REQUESTED** [review]

**Comment by Viou** [comment]
Updated!

**Review by Lurkars — APPROVED** [review]

**Pull request merged**